monasca-aggregator/Dockerfile

```diff
@@ -1,27 +1,54 @@
-from alpine:3.7
+from golang:1.10.3-alpine3.7 as builder
 
 arg AGGREGATOR_REPO=https://github.com/monasca/monasca-aggregator
 arg AGGREGATOR_BRANCH=master
 
+arg ALPINE_EDGE_REPO=http://uk.alpinelinux.org/alpine/edge/main
+
 env GOPATH=/go
 
-run apk add --no-cache openssl py2-jinja2 librdkafka && \
-  apk add --no-cache --virtual build-dep \
-    librdkafka-dev git go glide make g++ openssl-dev && \
-  mkdir -p $GOPATH/src/github.com/monasca/monasca-aggregator && \
+run apk add --no-cache openssl py2-jinja2 && \
+  apk add --no-cache --virtual build-dep git make g++ openssl-dev
+
+# librdkafka is not available in a stable alpine release
+run apk add --update --no-cache \
+        libressl \
+    --repository $ALPINE_EDGE_REPO  \
+    && apk add --update --no-cache \
+        librdkafka \
+        librdkafka-dev \
+    --repository $ALPINE_EDGE_REPO
```

Comment:
Previously here you used `edge/community` repo. `edge/main` is also OK? Do they have different versions of libraries?